pages/index.vue

```diff
@@ -1,9 +1,53 @@
 <template>
-  <NuxtLink :to="`/usecase_0/panel_1`">
-    Example use case
-  </NuxtLink>
+  <div class="flex flex-col w-screen h-screen bg-gray-200 p-2">
+    <!-- Banner -->
+    <div class="banner">
+      <Banner />
+    </div>
+
+    <div class="flex overflow-auto">
+      <!-- usecases -->
+      <div class="m-2 bg-white rounded">
+        <!-- category 1 -->
+        <h3 class="text-2xl">
+          Category 1
+        </h3>
+        <div class="flex text-left">
```

Comment:
This now adds one usecasecard *for each page* of a use case. I think we are aiming for one usecasecard per usecase (which could potentially include a carousel/image slider so as to incorporate all pages in that single card).

Reply:
Good idea! We will test/implement the carousel display in another PR.